src/components/RuntimeManager.js

```diff
@@ -103,6 +104,7 @@ const RuntimeErrorNotification = ({ runtime }) => {
 
   return h(Fragment, [
     h(Clickable, {
+      'aria-label': 'Runtime error notification',
```

Comment:
The clickable isn't the notifications itself, it's a button to see the log info, so I'd say the contained text is actually more helpful for someone using a screen reader, since this doesn't tell them what clicking it will do.
```suggestion
```